react_app/src/components/App/App.js

```diff
@@ -6,7 +6,7 @@ import ContactPage from "../ContactUs/ContactUs";
 import AboutPage from "../AboutPage/AboutPage";
 import LoginPage from "../LoginPage/LoginPage";
 import {
-    BrowserRouter as Router,
+    HashRouter as Router,
```

Comment:
see BrowserRouter vs HashRouter:
https://stackoverflow.com/questions/51974369/hashrouter-vs-browserrouter

Reply:
what was the reason for the switch? couldn't quite infer from the article.